scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -43,12 +46,13 @@
 axis_passMT = hist.axis.Boolean(name = "passMT")
 
 nominal_axes = [axis_eta, axis_pt, axis_charge, axis_passIso, axis_passMT]
+ptV_axis = hist.axis.Variable([0.0, 2.9, 4.7, 6.7, 9.0, 11.8, 15.3, 20.1, 27.2, 40.2, 13000.0], name="genPtV")
 
 # extra axes which can be used to label tensor_axes
 
-down_up_axis = hist.axis.Regular(2, -2., 2., underflow=False, overflow=False, name = "down-up variation")
+down_up_axis = hist.axis.Regular(2, -2., 2., underflow=False, overflow=False, name = "downUpVar")
 
-down_nom_up_axis = hist.axis.Regular(3, -1.5, 1.5, underflow=False, overflow=False, name = "down-nominal-up variation")
+down_nom_up_axis = hist.axis.Regular(3, -1.5, 1.5, underflow=False, overflow=False, name = "downUpNomVar")
```

Comment:
the name is potentially confusing since the actual ordering is down-nom-up

Reply:
Oh, yeah, then this is wrong

Reply:
Fixed